pyvista/plotting/plotting.py

```diff
@@ -167,17 +164,26 @@ class BasePlotter(PickingHelper, WidgetHelper):
     def __init__(self, shape=(1, 1), border=None, border_color='k',
                  border_width=2.0, title=None, splitting_position=None,
                  groups=None, row_weights=None, col_weights=None,
-                 lighting='light kit'):
+                 lighting='light kit', theme=None):
         """Initialize base plotter."""
         log.debug('BasePlotter init start')
-        self.image_transparent_background = rcParams['transparent_background']
+        if theme is None:
+            self._theme = pyvista.global_theme
```

Comment:
Shouldn't we also make a copy of the global theme during creation? This would be more in line with the "explicit theme passed" path.

Reply:
Good point.  Fixed in 1aeca3ae71567cf7404f8c66147e1bb8f0440190